### ISSUE TITLE
Server names

### ISSUE THREAD

**Issue by emopedia** [Minehut/Meta#933]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**

Website and In-game

**Describe the feature**

Allowing bigger server names such as 12 or 16 characters would allow users to be more creative with their server name without being limited to only 10 characters.

**How the feature is useful**

Basically what I said above. It allows for more creativity and other stuff.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by tarna** [comment]
dupe of #522 #684 #721

**Comment by emopedia** [comment]
oops. searched the wrong term

**Comment by GStudiosX2** [comment]
> oops. searched the wrong term. someone can close this

pretty sure you can close it lol
